tests/multigrid-global-coarsening/mg_transfer_p_06.with_p4est=true.mpirun=1.output

```diff
@@ -0,0 +1,271 @@
+
+DEAL:0:CG<2>(1)<->CG<2>(1):active::3.8729833
+DEAL:0:CG<2>(1)<->CG<2>(1):active::1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 
+DEAL:0:CG<2>(1)<->CG<2>(1):active::1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 
+DEAL:0:CG<2>(1)<->CG<2>(1):active::6.4420494
+DEAL:0:CG<2>(1)<->CG<2>(1):active::1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 1.0000000 
+DEAL:0:CG<2>(1)<->CG<2>(1):active::1.0000000 1.0000000 1.0000000 1.0000000 1.5000000 2.0000000 0.50000000 1.0000000 0.50000000 1.0000000 1.0000000 1.0000000 2.5000000 1.5000000 1.5000000 1.0000000 1.0000000 2.0000000 1.0000000 1.0000000 1.0000000 1.0000000 2.0000000 1.0000000 1.0000000 
```

Comment:
To me the output does not look correct.

Coult you clear the content of:

https://github.com/dealii/dealii/blob/3e4283dc79386633545e791b09cc497912256448/include/deal.II/multigrid/mg_transfer_global_coarsening.templates.h#L2846-L2848